packages/patternfly-4/react-core/src/demos/PageLayout/examples/PageLayoutDefaultNav.js

```diff
@@ -203,7 +203,7 @@ class PageLayoutDefaultNav extends React.Component {
             </TextContent>
           </PageSection>
           <PageSection>
-            <Gallery gutter="md">
+            <Gallery gutter>
```

Comment:
boolean props should start with some kind of identifying prefix, e.g. useGutter or hasGutter

Reply:
I like `has*`. `use*` could be confused with React hooks api parts. That said, maybe there is a distinction to be made between boolean props that have a presentational impact only vs behavioral impact? What about `usePf*`?

`usePfGutter` is pretty readable and wouldn't likely conflict with other prefixes. Thoughts?

Reply:
not sure about `usePf` but anyways, `hasGutter` would work fine I think